content/en/docs/tasks/administer-cluster/controller-manager-leader-migration.md

```diff
@@ -69,6 +68,22 @@ controllerLeaders:
     component: kube-controller-manager
 ```
 
+Alternatively, because the controllers can run under either controller managers, setting `component` to `*` for both side makes the configuration file consistent between both parties of the migration.
```

Comment:
(nit)
```suggestion
Alternatively, because the controllers can run under either controller managers, setting `component` to `*`
for both sides makes the configuration file consistent between both parties of the migration.
```